src/BuildScriptGenerator/Python/PythonBashBuildSnippet.sh.tpl

```diff
@@ -104,6 +104,18 @@ fi
 	echo $APP_PACKAGES_PATH > $SITE_PACKAGES_PATH"/oryx.pth"
 {{ end }}
 
+{{ if RunPythonPackageCommand }}
```

Comment:
Shouldn't this be mutually exclusive with the other things that we do in this tpl file? should we move this package stuff to a separate file if it improves readability?

Reply:
I thought about it .. but then I did a quick referrence in node .. and seems like it's in the same file .. so I keep it same